src/scripts/dev-app.sh

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env bash
```

Comment:
There is a `run-local.sh` script in bulk-fhir that allows different configs to be added. Thats probably a better script for this than the old `dev-app.sh`.